week-4/implement/count.test.js

```diff
@@ -15,3 +15,23 @@
 // And a character char that does not exist within the case-sensitive str,
 // When the function is called with these inputs,
 // Then it should return 0, indicating that no occurrences of the char were found in the case-sensitive str.
+function countOccurrences(input, char) {
+    const num = input.split(char).length - 1 
+    if(num => 1)
+    return `${char} occurs ${num} times`;
+else{`${char} occurs 0 times`;
```

Comment:
Make sure that Prettier is installed and working in Visual Studio Code. This would help to keep formatting consistent, especially around `if` blocks